src/curated-corpus/components/ApprovedItemCurationHistory/ApprovedItemCurationHistory.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { ApprovedCorpusItem } from '../../../api/generatedTypes';
+import { ScheduleHistoryEntries } from '../ScheduleHistoryEntries/ScheduleHistoryEntries';
+import {
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Typography,
+} from '@material-ui/core';
+import FaceIcon from '@material-ui/icons/Face';
+import ThumbUpIcon from '@material-ui/icons/ThumbUp';
+import { getCuratorNameFromLdap } from '../../helpers/helperFunctions';
+import { DateTime } from 'luxon';
+
+interface ApprovedItemCurationHistoryProps {
+  /**
+   *
+   */
+  item: ApprovedCorpusItem;
+}
```

Comment:
I think you forgot to fill in that comment block

Reply:
How. Embarrasing. All of these comments are to read 🤣. Especially the `console.log()` one! Thanks heaps, @Herraj! :)